README.md

```diff
@@ -4,7 +4,55 @@
 [![Dashboard badcamp-2019](https://img.shields.io/badge/dashboard-badcamp_2019-yellow.svg)](https://dashboard.pantheon.io/sites/77e169f6-098e-4e68-b9b1-67d0b7cc2f02#dev/code)
 [![Dev Site badcamp-2019](https://img.shields.io/badge/site-badcamp_2019-blue.svg)](http://dev-badcamp-2019.pantheonsite.io/)
 
-
 ## Designs
-* [Zeplin](https://app.zeplin.io/project/5c9a927bca007d05b8951ac2/dashboard) 
-* [Sketch](https://sketch.cloud/s/q9j5d)
+
+- [Zeplin](https://app.zeplin.io/project/5c9a927bca007d05b8951ac2/dashboard)
+- [Sketch](https://sketch.cloud/s/q9j5d)
+
+## Bootstrapping Your Dev Environment
+
+You have your choice of local environments, but since we have build-in support for Docksel, Docksal is the quickest way to get started.  Getting ddev, Lando or similar is an exercise for the Reader :-) We welcome documentation for other environments if anyone wants to contribute these. 
+
+Steps:
+
+- [Install Docksal for your platform] (https://docksal.io/installation/).
+
+- Use git to pull the website repo (https://github.com/badcamp/badcamp-2019)
+
+- In your Pantheon account (you'll want to create one if you don't have one already) you'll need to create a token for a secure link between your install and the Pantheon dev and test environments.  Once you do this, you should talk to Sean or some other designated adult, who will "bless" your Pantheon account into the BADCamp dev group.
+
+- Add settings to your badcamp-2019 install. These go in your `.docksal` directory at the base of the repo. The file is called `docksal-local.env` and looks like this on my install:  
+```
+SECRET_TERMINUS_TOKEN="YOUR-PANTHEON-TOKEN-HERE"
+NGINX_DRUPAL_FILE_PROXY_URL=https://dev-badcamp-2019.pantheonsite.io/
```

Comment:
@torenware this doesn't need to go in the `.docksal/docksal-local.env` file if anything it should actually be fixed in the `.docksal/docksal.env` file to read 

```
NGINX_DRUPAL_FILE_PROXY_URL="https://${PANTHEON_ENV}-${PANTHEON_SITE}.pantheonsite.io/"
```

Reply:
@sean-e-dietrich  sounds reasonable. I'll do the NGINX_DRUPAL_FILE_PROXY_URL as a separate PR, and fix the text here.